src/components/crossdomains/linkShopCrossDomain.ts

```diff
@@ -58,3 +60,9 @@ export default zoid.create({
     },
   },
 });
+
+// eslint-disable-next-line
+// @ts-ignore
+window.psAccountZoidExport = zoidExport;
```

Comment:
You can avoid using ts-ignore if you add psAccountZoidExport to the window interface, you can do that by adding a window.d.ts file 
```ts
declare global {
  interface Window {
    psAccountZoidExport: any // Use a type if there is one 
  }
}

export {};
```

Reply:
Added it to the shims-global.d.ts